openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -11,6 +11,7 @@
 
 from django.utils import timezone
 from django.utils.translation import ugettext as _
+from django.utils.translation import ungettext
```

Comment:
It took me 3 passes to realize this wasn't ugettext. I'm guessing this is for dealing with plurarlity?

Reply:
Yeah. It's unicode version of ngettext. Which is actually the default in python3 anyway. So maybe I'll switch these both to `gettext` and `ngettext` for a little bit extra clarity.

But yeah, ngettext is for plurals.